assets/Documenter.css

```diff
@@ -5,3 +5,16 @@ div.wy-menu-vertical ul.current li.toctree-l3 a {
 a.documenter-source {
   float: right;
 }
+
+/* .methodtable */
+.methodtable pre {
```

Comment:
Prefix class with `documenter-`? That's what I've done with `source` above, but if you don't feel it'll be needed feel free to change that one as well.